README.md

```diff
@@ -568,6 +569,35 @@ convert ckl2POAM              Translate DISA Checklist CKL file(s) to POA&M file
   EXAMPLES
     $ saf convert ckl2POAM -i checklist_file.ckl -o output-folder -d abcdefg -s 2
 ```
+
+[top](#convert-other-formats-to-hdf)
+#### CycloneDX SBOM to HDF
```

Comment:
explicitly mention the formats that worked for us from the example list.  ex. sbom, vex, i think the hardware one worked.  and then explicitly say which ones don't work, ex. the SaaS one.

Reply:
Addressed.